CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#20)
 
 New features:
 
 Bugfixes:
+- Changed `Foldable1`'s `fold1` member to `foldl1` and `foldr1` (#21, #23)
```

Comment:
Not sure if this is the correct summary. Also not sure if this is considered a bugfix.

Reply:
Yeah.... Not sure myself either. It's not a breaking change. I wouldn't say it's a bug fix either though because there wasn't a release that included the unchanged instance, so that it no longer worked and we needed to fix it in a separate release. 

I think I would move this to "Other improvements" and say something like 
> Updated `Identity`'s `Foldable1` instance to account for changes in `Foldable1` definition.